api/api/http.js

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
Add `PATCH`. We support it in some APIs, e.g. https://opensearch.org/docs/2.4/security/access-control/api/#patch-action-group